src/audio/copier/copier.c

```diff
@@ -591,7 +591,7 @@ static int copier_process(struct processing_module *mod,
 	switch (dev->ipc_config.type) {
 	case SOF_COMP_HOST:
 		if (!cd->ipc_gtw)
-			return host_common_copy(cd->hd, dev, copier_host_dma_cb);
+			return cd->hd->copy(cd->hd, dev, copier_host_dma_cb);
```

Comment:
nak, we cant kill wrappers.

Reply:
if can't kill, how about inline? inline have same effect with this change.

Reply:
yes - we can inline and should inline.